tests/fixtures.py

```diff
@@ -0,0 +1,28 @@
+from cms.api import create_page
+
+
+class B4TestFixture(object):
+    """Sets up generic setUp and tearDown methods for tests."""
+
+    def setUp(self):
+        self.language = "en"
+        self.home = create_page(
+            title="home",
+            template="page.html",
+            language=self.language,
+        )
+        self.home.publish(self.language)
+        self.page = create_page(
+            title="content",
+            template="page.html",
+            language=self.language,
+        )
+        self.page.publish(self.language)
+        self.placeholder = self.page.placeholders.get(slot="content")
+        self.superuser = self.get_superuser()
+        self.request_url = self.page.get_absolute_url(self.language) + "?toolbar_off=true"
```

Comment:
Hmm, maybe a good idea to call `super()`? In case it gets mixed into a more elaborate `TestCase` hierarchy at some point.